Assets/Scripts/Game/Leveling/AlignmentSystem.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.Events;
+
+namespace RPG
+{
+    public class AlignmentSystem : MonoBehaviour
+    {
+        public UnityEvent<int> onAligneChanged;
+
+        public int Sexiness
```

Comment:
These should probably be floats since I think we decided on a -1.0 to +1.0 scale for the alignment axes. That way we can make any scale of changes.